core/math/math_funcs.h

```diff
@@ -187,6 +187,33 @@ class Math {
 #endif
 	}
 
+	/**
+	 * @note Assumes (p_num + p_den) doesn't overflow
```

Comment:
This should be a single line, and no "@note", we don't have yay s:
```suggestion
	 // Assumes (p_num + p_den) doesn't overflow.
```